languages/sr_RS/Zend_Validate.php

```diff
@@ -0,0 +1,308 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * sr_RS-Revision: 28.October.2015
+ */
+return array(
+    // Zend\Authentication\Validator\Authentication
+    "Invalid identity" => "Погрешан идентитет",
+    "Identity is ambiguous" => "Идентитет је сумњив",
+    "Invalid password" => "Погрешна лозинка",
+    "Authentication failed" => "Неуспела аутентификација",
+
+    // Zend\I18n\Validator\Alnum
+    "Invalid type given. String, integer or float expected" => "Погрешан тип унет. Очекиван је текст, цео број или стварни број",
```

Comment:
I'd rather use native english words which are already known by the developers: `string`, `integer`, `float`. I think this phrase is used in validation context so it just makes more sense to use _programming_ terms.